### PR TITLE
Working on #2972 Template Conversion to Twig Format (error)

### PR DESCRIPTION
**Pull request by Shade-**
For #2972. Templates converted:
- [x]  error
- [x]  error_nopermission
- [x]  error_nopermission_loggedin
- [x]  error_inline

Templates not used by MyBB and marked to deletion:
- [x]  error_attacherror

### CODE REVIEW

File: inc/views/base/error/error.twig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{% if title %}{{ title }}{% else %}{{ mybb.settings.bbname }}{% endif %}</title>
+{% endblock head %}
+
+{% block body %}
+<table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+    <tr>
+        <td class="thead"><span class="smalltext"><strong>{% if title %}{{ title }}{% else %}{{ mybb.settings.bbname }}{% endif %}</strong></span></td>
```
Comment (euantorano) [inline inc/views/base/error/error.twig]: Personally I'd prefer to have this spread across multiple lines, like:

```twig
{% if title %}
    {{ title }}
{% else %}
    {{ mybb.settings.bbname }}
{% endif %}
```

I don't mind things all in one line if it's not too long, but this is quite a long line to read.

File: inc/views/base/error/no_permission.twig [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+</ol>
+
+{% if mybb.user.uid > 0 %}
+    <br />
+    {{ trans('error_nopermission_user_username', mybb.user.username) }}
+{% else %}
+
+    <form action="member.php" method="post">
+        <input type="hidden" name="action" value="do_login" />
+        <input type="hidden" name="url" value="{{ redirect_url }}" />
```
Comment (euantorano) [inline inc/views/base/error/no_permission.twig]: I'm not 100% sure, but we should maybe be using `|e('url')` for escaping URLs: https://twig.symfony.com/doc/2.x/filters/escape.html - we need to test it and see what it does, and if that's the case, we need to check existing translations.

**Comment by Shade-** [comment]
Ouch. Might redo this completely.

**Comment by Shade-** [comment]
Conflicts resolved.

**Comment by euantorano** [comment]
Yay!

**Pull request merged**